# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| Groep: XX     |  |
| :-------------|:-------------|
| Davante Scheers |  | 6580599 |
| Bob Villerius | 6483895 |
| Daan d' Engelbronner| 6537812 |
| Planning Groep: 42     | 10:45 / 17:30  |
|---|---|
| Sanity Check | 10:45 - 11:45 |
| Condensator Bouwen | 11:45 - 12:15 |
| Condensator Testen/Meten | 12:15 - 12:30 |
| Pauze 1 | 12:30 - 13:45 |
| Synthese | 13:45 - 14:15 |
| Sensor Ontwerpen | 14:15 - 14:30 |
| Sensor Bouwen | 14:30 - 15:30 |
| Pauze 2 | 15:30 - 15:45 |
| Sensor Data Verwerken | 15:45 - 16:30 |
| Controleren / Afvinken / Opruimen | 16:30 - 17:30 |

## *Opdracht 1*: Sanity check 3

#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.
<img src="FotoSchetsCondensator.jpeg" alt="schets student 1" width="400"/>

A staart hierin als 0.01 m^2

d staat hierin als ongveer 1 millimeter = 0.001 m

In [ ]:
import numpy as np
import matplotlib.pyplot as plt

epsilon0 = 8.854e-12  # permittiviteit van vacuum (F/m)

# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!

def vlakkeplaat(A,d, epsilon):
    C = epsilon * A/d
    return C

# Optioneel: schrijf je functie zodanig dat je arrays als input kan geven en (dus) ook arrays als output.


In [25]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren
# gaat zijn.

epsilon_polypropyleen = 2.2 * epsilon0
d = 0.001
A = 0.1**2
C_berekend = vlakkeplaat(A, d, epsilon_polypropyleen)
print(C_berekend / 10**-12, "pF. Dit is de  berekende capaciteit van onze plaatcondensator.")

# C = [functieNaam(2.0), functieNaam(4.7),functieNaam(1.2)]


# print(f"Berekende capaciteit plaatcondensator groep XX is {C[0]:.1f} pF")
C_gemeten = 1.7/(1000*8*2*np.pi*100000)
print(C_gemeten / 10**-12, "pF. Dit is de gemeten capaciteit van onze plaatcondensator.")


194.788 pF. Dit is de  berekende capaciteit van onze plaatcondensator.
338.20425407027767 pF. Dit is de gemeten capaciteit van onze plaatcondensator.


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
![scope meting 1](FotoCondensatorOpstelling.jpeg)
![scope meting 2](FotoCondensatorMeting.jpeg)

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit   | Berekend Voltage | Gemeten Voltage | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| groep 42| 200pF | 1.34 V | 1.7V | 338 pF |

Het gemeten voltage/capaciteit is hoger dan onze berekende waarde, dit komt waarschijnlijk doordat we voor d een schatting hebben gemaakt die waarschijnlijk boven de daadwerkelijke waarde van d zit, waardoor de meting hoger uitkomt dan onze berekening.

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.
![drie schetsen](tudelftLogo.png)


## Opdracht 3: Welke sensor gaan jullie maken?

Pas deze tekst aan: Wij gaan een sensor maken die X meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [3]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.

## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [4]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [5]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.